metadata/boundaries/county_boundaries

```diff
@@ -0,0 +1,74 @@
+# Title
+
+Utah County Boundaries
+
+## ID
+
+b2c8af56-3d92-4aa7-ac2f-b1388051d9ce
+
+## Brief Summary
+
+A statewide polygon dataset of county boundaries for cartography and approximate boundary identification.
+
+## Summary
+
+This dataset contains polygons for all 29 counties in Utah, where the perimeter of the polygon represents the county boundary. Each polygon also contains county population as of the [2020 Census](https://www.census.gov/library/stories/state-by-state/utah-population-change-between-census-decade.html) and current approximate population.
+
+## Description
+
+### What is the dataset
+
+This dataset represents each county as a polygon with information such as the Entity Number and FIPS code. A full explanation of the field definitions for this dataset is available on the [schema summary](https://docs.google.com/document/d/1e98nKP6PW9tuTy3nZYS4qhMp9fC8E6uv73SVbBOYvsk/edit?usp=sharing) document.
+
+### What is the purpose of the dataset
+
+This dataset provides an administrative baseline for the current extent of each county in Utah. Boundary data is often used in analysis, especially within the transportation, real estate, and natural resources sectors.
+
+### What does the dataset represent
+
+This dataset does not represent exact legal boundaries, but, rather a set of boundaries used for general purposes that conforms to logical and administrative rules (e.g. no two cities or township areas may cover the same geographic extent).
+
+### How was the dataset created
+
+UGRC developed this dataset using coordinate geometry [(COGO)](https://pro.arcgis.com/en/pro-app/latest/help/editing/introduction-to-cogo.htm) from legal descriptions provided by each county. When necessary, this dataset was adjusted to conform to known physical features such as rivers or mountain ranges.
+
+All updates to this dataset must first be filed with the [Lieutenant Governor's](https://ltgovernor.utah.gov/) office. Once approved, UGRC receives updated data and adjusts the polygon boundaries accordingly.
+
+### How reliable and accurate is the dataset
+
+This dataset reflects the most current and complete version of Utah county boundaries. Modifications proposed by counties are first certified through the Lieutenant Governor's Office, then applied to this dataset by UGRC.
+
+## Credits
+
+### Data Source
+
+- UGRC
```

Comment:
```suggestion
- Lieutenant Governor's Office
```

Since we just maintain the gis representation of data from the Lt Gov, I would replace us with the LGO here.